src/modules/addcomment.js

```diff
@@ -0,0 +1,23 @@
+const keepComment = () => {
+  const name = document.querySelector('#name');
+  const comment = document.querySelector('#msg');
+  const commentId = document.querySelector('#commentId');
+  const nameInput = name.value;
+  const commentInput = comment.value;
+  const commentIdval = commentId.value;
+  fetch(`https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/UeRVpttbuH5idOpNpTZz/comments?item_id=${commentIdval}`, {
+    method: 'POST',
+    body: JSON.stringify({
+      item_id: commentIdval,
+      username: nameInput,
+      comment: commentInput,
+    }),
+    headers: {
+      'Content-Type': 'application/json',
+    },
+  });
+  name.value = '';
+  comment.value = '';
+};
```

Comment:
- [Required] I think since this function sends some data to an `API` online and it might take some time it would be better if you use `async` and `await` with the `fetch` method to send the data. The keyword `async` before a function makes the function return a promise and the `await` keyword makes the function pause the execution and wait for a resolved promise before it continues which in turn makes your function run synchronously without an error. Please consider this change and rectify it.

## Example code 
```JavaScript
const keepComment = async () => {
  const name = document.querySelector('#name');
  const comment = document.querySelector('#msg');
  const commentId = document.querySelector('#commentId');
  const nameInput = name.value;
  const commentInput = comment.value;
  const commentIdval = commentId.value;
  await fetch(`https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/UeRVpttbuH5idOpNpTZz/comments?item_id=${commentIdval}`, {
    method: 'POST',
    body: JSON.stringify({
      item_id: commentIdval,
      username: nameInput,
      comment: commentInput,
    }),
    headers: {
      'Content-Type': 'application/json',
    },
  });
  name.value = '';
  comment.value = '';
} 
```